packages/vue/src/view/Application.vue

```diff
@@ -0,0 +1,15 @@
+<template>
+  <Layout>
```

Comment:
In case of vue you need to use component name lowercase and separated with hyphen